### DISCUSSION TITLE
中国区域链接卡顿

### DISCUSSION THREAD

**Discussion by kongxia114** [community/community#29554] [Other Feature Feedback, Questions, & Ideas]
GitHub 官方是否能添加中国网络节点

**Comment by mazkdevf** [comment]
China -> English

官方是否能添加中国网络节点 = Can the official add Chinese network nodes?

**Comment by mazkdevf** [comment]
> GitHub 官方是否能添加中国网络节点

What do you mean? - 你是什​​么意思？

Google translate ^^
